pkg/commands/options/build.go

```diff
@@ -31,8 +31,8 @@ import (
 )
 
 const (
-	// configDefaultBaseImage is the default base image if not specified in .ko.yaml.
-	configDefaultBaseImage = "distroless.dev/static:latest"
+	// ConfigDefaultBaseImage is the default base image if not specified in .ko.yaml.
+	ConfigDefaultBaseImage = "cgr.dev/chainguard/static:latest"
```

Comment:
If it's going to be used externally, `pkg/build.DefaultBaseImage` is better? I don't think I want to recommend other Go code depend on `pkg/commands`. Maybe we should make it `internal/commands` instead to prevent that?